R/cache.R

```diff
@@ -286,7 +290,7 @@ uncached <- function(fn) {
   }
 }
 
-build_cached_function <- function(cached_function, safe_columns) {
+build_cached_function <- function(cached_function, safe_columns, blacklist) {
```

Comment:
This is called argument bloat. We should stick with `cached_function` as the sole formal.